tools/pybind11Tools.cmake

```diff
@@ -218,8 +220,10 @@ function(pybind11_add_module target_name)
   # Make sure C++11/14 are enabled
   if(PYBIND11_CPP_STANDARD)
     if(CMAKE_VERSION VERSION_LESS 3.3 OR CMAKE_GENERATOR MATCHES "Visual Studio")
+      message("HERE, I hope?")
       target_compile_options(${target_name} PUBLIC ${PYBIND11_CPP_STANDARD})
     else()
+      message("NOT HERE, I hope?")
```

Comment:
Have you left these messages in by intent ? (on line 223, too)

Reply:
Yes. This is in the commit with message "Debugging non-failure of /std:c++17", and this PR is not ready, yet.

Reply:
Yes. This is in the commit with message "Debugging non-failure of /std:c++17", and this PR is not ready, yet.